### ISSUE TITLE
[Issue]: The "replace preview" button stopped including prompt exif information in the image

### ISSUE THREAD

**Issue by crazygonuts** [vladmandic/automatic#1457]
### Issue Description

This bug is also happening with a1111 as reported here:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/11163

using the "replace preview" button saves a version of the image with no embedded metadata.  Using the image from the output directory or saving the image from the generation box will give you an image with the correct metadata.

### Version Platform Description

Version: 5808b7a6 Sat Jun 17 08:13:19 2023 -0400
chrome
windows 10

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by vladmandic** [comment]
correct. those images go through separate and simplified save process and do not have access to metadata at that time.
it could be done, but i do not see it as desired as preview images are just that - previews.

however, i will allow for contributions if someone wants to prove me wrong - so i'm converting this into feature request.
its definitely not an issue.

**Comment by crazygonuts** [comment]
It's more useful to have the metadata used to generate preview images for a variety of reasons - comparing models, reference for style or quality modifiers, etc.

additionally, as the author of the bug in the a1111 issue points out, it used to save the metadata when saving previews until sometime recently.  I have 500+ models with preview images, most of which were saved using "replace preview" as that was the easier and quicker than manually copying and renaming them, and except for the ones created recently they all have metadata.

**Comment by vladmandic** [comment]
sure, that's all valid and i'm open to contributions. right now i don't see this anyime soon on my backlog.

**Comment by vladmandic** [comment]
done.
